content/applications/marketing/surveys/overview/time_random.rst

```diff
@@ -1,34 +1,49 @@
-============================
-Time and Randomize Questions
-============================
+==============================
+Timed and randomized questions
+==============================
 
-Time Limit
+When creating a survey in Odoo, the user has the options to set a time limit on the survey and
+randomize the questions.
+
+Time limit
 ==========
 
-| On timed surveys, respondents need to complete the survey within a certain period of time. It can
-  be used to ensure that all respondents get the same amount of time to find the answers, or to
-  decrease the chance of having them looking at external resources.
-| Set the *Time limit* under the tab *Options*.
+During a timed survey, participants must finish the survey within a certain period of time. Placing
+a time limit on the survey ensures that every participant gets the same amount of time to fill in
+answers. It also greatly reduces the chance of participants looking up responses via external
+resources (Google, etc.).
+
+The :guilabel:` Survey Time Limit` setting is found in the :guilabel:`Options` tab of the survey
+form under the :guilabel:`Questions` section.
 
-.. image:: time_random/tab_options_timelimit.png
+.. image:: time_random/time-limit.png
    :align: center
-   :alt: View of a survey form emphasizing the time limit feature in Odoo Surveys
+   :alt: Time limit field in the options tab of a survey template form.
+
+When a time limit is configured, a timer is displayed on every page of the survey, letting
+participants keep track of the time remaining. Participants that do not submit their survey by the
+preconfigured time limit will *not* have their answers saved.
+
+Randomized selection
+====================
+
+When a survey is randomized, Odoo shuffles the questions and reveals them in a random order every
+time a participant begins the questionnaire.
 
-A timer is shown on the pages so the user can keep track of the remaining time. Surveys not \
-submitted by the *Time limit* do not have their answers saved.
+Randomizing surveys is an effective way to discourage participants from looking at each other's
+responses.
```

Comment:
Optional:
Would condense the two sentences together since they link to the same idea.
Suggested rewrite for second sentence for more context around use case.
```suggestion
time a participant begins the questionnaire. Using randomization as a survey method discourages
participants from looking at each other's responses, and helps control for individual testing.
```